crates/fluvio-controlplane-metadata/src/smartmodule/package.rs

```diff
@@ -242,6 +265,29 @@ impl Decoder for FluvioSemVersion {
     }
 }
 
+impl std::fmt::Display for SmartModuleVisibility {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::result::Result<(), std::fmt::Error> {
+        let lbl = match self {
+            Self::Private => "private",
+            Self::Public => "public",
+        };
+        write!(f, "{}", lbl)?;
+        Ok(())
```

Comment:
I think we can do:

```suggestion
        write!(f, "{}", lbl)
```

Here instead, wdyt?

Reply:
good catch! I agree